pages/book/security-best-practices.mdx

```diff
@@ -0,0 +1,144 @@
+# Security Best Practices
```

Comment:
Some of the examples below demonstrate "harmful actions", while others describe "how-to's". I suggest that there would be another level of headings, and all bad things would be listed under `## Common pitfalls`, while all good things would be listed under `## Best practices`. All the current h2 (`##`) headings would become h3 (`###`) ones.

Alternatively, you might consider combining both under "best practices", since that's the title of the page. In this case, please, name the headings in a positive way (`## Use or do this` over `## Bad action not to perform`) and also provide "valid examples" ("recommendations") whenever possible :)